src/test/groovy/de/tracetronic/jenkins/plugins/ecutestexecution/steps/RunPackageStepIT.groovy

```diff
@@ -210,22 +209,52 @@ class RunPackageStepIT extends IntegrationTestBase {
 
     }
 
-    def 'Run pipeline: timeout by busy ecu.test'() {
+    def 'Run pipeline: with 409 handling and config load'() {
         given:
             GroovyMock(RestApiClientFactory, global: true)
             def restApiClient =  new RestApiClientV2('','')
             RestApiClientFactory.getRestApiClient(*_) >> restApiClient
+
             def mockCall = Mock(Call)
             mockCall.clone() >> mockCall
-            mockCall.execute() >> MockApiResponse.getResponseBusy()
-            GroovySpy(ConfigurationApi, global: true){
-                manageConfigurationWithHttpInfo(*_) >> {
-                    restApiClient.apiClient.execute(mockCall, null)
+            mockCall.execute() >> MockApiResponse.getResponseBusy() >> MockApiResponse.getResponseUnauthorized()
+
+            def manageConfigCalled = 0
+            GroovySpy(ConfigurationApi, global: true) {
+                manageConfiguration(*_) >> {
+                    manageConfigCalled =+ 1
+                    restApiClient.apiClient.execute(mockCall, new TypeToken<SimpleMessage>(){}.getType())
                 }
             }
+
+            WorkflowJob job = jenkins.createProject(WorkflowJob.class, 'pipeline')
+            job.setDefinition(new CpsFlowDefinition("node { ttRunPackage testCasePath: 'test.pkg', testConfig: [tbcPath: 'test.tbc'] }", true))
+        when:
+            WorkflowRun run = jenkins.assertBuildStatus(Result.FAILURE, job.scheduleBuild2(0).get())
+        then:
+            jenkins.assertLogContains("Executing package 'test.pkg'", run)
+            jenkins.assertLogNotContains('ecu.test is busy', run)
+            jenkins.assertLogContains('unauthorized', run)
+        and:
+            // cannot use interactive based testing since the method call was not fully ended
```

Comment:
sounds a bit strange
"interactive-based testing cannot be used because the method call does not fully end."
or
"interactive-based testing cannot be used because the method call does not fully complete."